lib/get-config.js

```diff
@@ -268,7 +269,7 @@ function processExtends(config, options) {
           delete configuration.loadedConfigurations;
 
           if (configuration.rules) {
-            config.rules = Object.assign({}, config.rules, configuration.rules);
+            extendedRules = Object.assign({}, extendedRules, configuration.rules);
```

Comment:
Am I missing something? Why don't we just do

```suggestion
            extendedRules = Object.assign({}, configuration.rules);
```

Reply:
I did miss something. It's looping.